### DISCUSSION TITLE
[HELP]

### DISCUSSION THREAD

**Discussion by gonzalezharry** [gtk-rs/gtk4-rs#1602] [Q&A]
error: process didn't exit successfully: `target\debug\main.exe` (exit code: 0xc0000139, STATUS_ENTRYPOINT_NOT_FOUND)

This happens with gtk4 only, gtk worked perfectly.

I installed gtk4 with MSYS2 and also witth vcpkg for the stable-msvc toolchain.

Getting the same error both with gnu and msvc.

All path directories included, ran dependecy walker and couldn't manage to find what's wrong.

**Comment by KairunH** [comment]
Put the bin and everything with gtk in (env path) to the very top.
